cloudant-client/src/main/java/com/cloudant/client/api/views/ViewResponse.java

```diff
@@ -111,6 +121,14 @@
          * @since 2.0.0
          */
         <D> D getDocumentAsType(Class<D> docType);
+
+        /**
+         * Gets a map of the document id and error message for this row.
+         *
+         * @return a map with an entry for key of the doc id and value of error
+         * @since 2.10.0
+         */
+        Map<String, String> getError();
```

Comment:
I'd prefer this to be `String getError()` to avoid forcing a binding of the error to the ID, which can be returned independently by `getId()` anyway.